src/components/account/Quota.vue

```diff
@@ -0,0 +1,109 @@
+<!--
+  - SPDX-FileCopyrightText: 2023 Nextcloud GmbH and Nextcloud contributors
+  - SPDX-FileCopyrightText: 2024 STRATO AG
+  - SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+
+<template>
+	<section class="section">
+		<h2>{{ t('simplesettings', 'Storage usage') }}</h2>
+		<div class="quota">
+			<NcProgressBar
+				v-if="hasLimitedSpace"
+				size="medium"
+				:value="usageRelative"
+				:color="barColor"
+				:style="{'background-color': backgroundColor}" />
+			<div class="quota-info">
+				<p>
+					<span v-if="hasLimitedSpace" :style="quotaPillStyles(barColor)" />
+					<span v-html="quotaUsedText" />
+				</p>
+				<p v-if="hasLimitedSpace">
+					<span :style="quotaPillStyles(backgroundColor)" />
+					<span v-html="quotaFreeText" />
+				</p>
```

Comment:
todo: we should avoid using `v-html` because of the risk if XSS attacks
```suggestion
				<p>
					<span v-if="hasLimitedSpace" :style="quotaPillStyles(barColor)" />
					<span>
						{{ quotaUsedPrefix }}<strong>{{ usage }}</strong>
					</span>
				</p>
				<p v-if="hasLimitedSpace">
					<span :style="quotaPillStyles(backgroundColor)" />
					<span>
						{{ quotaFreePrefix }}<strong>{{ freeSpace }}</strong>
					</span>
				</p>
```

provide the values to the template (l.43)
```vue
	data() {
		return {
			usageRelative,
			barColor,
			backgroundColor,
			hasLimitedSpace,
			freeSpace,
			usage,
		}
	},
```

Reply:
✅